src/runChromaticBuild.test.ts

```diff
@@ -0,0 +1,79 @@
+// eslint-disable-next-line import/no-unresolved
+import * as chromatic from "chromatic/node";
+
+import { runChromaticBuild } from "./runChromaticBuild";
+
+jest.mock("chromatic/node", () => ({ run: jest.fn() }));
```

Comment:
```suggestion
import { run } from "chromatic/node";

jest.mock("chromatic/node");
const mockedRun = mocked(run);
```

Reply:
If only that would work. I tried (and just tried again).